internal/source/prometheus/source.go

```diff
@@ -0,0 +1,75 @@
+package prometheus
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/kubeshop/botkube/pkg/api"
+	"github.com/kubeshop/botkube/pkg/api/source"
+)
+
+const (
+	// PluginName is the name of the Prometheus Botkube plugin.
+	PluginName = "prometheus"
+)
+
+// Source prometheus source plugin data structure
+type Source struct {
+	prometheus    *Client
+	l             sync.Mutex
+	pluginVersion string
+	startedAt     time.Time
+}
+
+func NewSource(version string) *Source {
```

Comment:
```suggestion
// NewSource returns a new instance of Source.
func NewSource(version string) *Source {
```

Just for the sake of having comments on public methods 🙂